### PR TITLE
Adding watch process.

### PR DESCRIPTION
**Pull request by tomgenoni**

### CODE REVIEW

**Comment by tomgenoni** [comment]
@danoc please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 ## [Unreleased][unreleased]
 
 ## [4.0.1][4.0.1] - 2015-09-22
+### Added
```
Comment (danoc) [inline CHANGELOG.md]: This is under the wrong section.

File: gulpfile.js [hunk 2]
```diff
@@ -71,6 +71,11 @@ gulp.task('sass', function() {
     .pipe(gulp.dest(paths.css));
 });
 
+// Watch tasks
+gulp.task('watch', function() {
+    gulp.watch('src/core/**/*.scss', ['sass']);
```
Comment (danoc) [inline gulpfile.js]: This should be two spaces.

**Comment by danoc** [comment]
Two small changes then LGTM.

**Pull request merged**